From: chasey-dev <ellenyoung0912@gmail.com>
Subject: [PATCH] mt_wifi_osal: Include RPS declarations

RPS declarations are provided by net/rps.h in newer kernels. Include
that header under the corresponding kernel version condition.

Signed-off-by: chasey-dev <ellenyoung0912@gmail.com>
--- a/mt_osal_packet.c
+++ b/mt_osal_packet.c
@@ -25,6 +25,9 @@
 #if (KERNEL_VERSION(6, 6, 0) <= LINUX_VERSION_CODE)
 #include <net/netdev_rx_queue.h>
 #endif
+#if (KERNEL_VERSION(6, 9, 0) <= LINUX_VERSION_CODE)
+#include <net/rps.h>
+#endif
 
 #define SUBMODULE "mt_osal_packet"
 #include <mt_osal_debug.h>
